AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/States/AuthState.swift

```diff
@@ -19,6 +19,8 @@ enum AuthState: State {
 
     case configured(AuthenticationState, AuthorizationState)
 
+   // case configurationError(Error)
+
```

Comment:
If not being used, should we remove this?

Reply:
removed